diesel/src/query_builder/select_statement/dsl_impls.rs

```diff
@@ -196,3 +200,58 @@ impl<'a, S, F, D, W, O, L, Of, G, DB> InternalBoxedDsl<'a, DB>
         )
     }
 }
+
+impl<'a, F, D, W, O, L, Of, G, DB> InternalBoxedDsl<'a, DB>
```

Comment:
Yeah, I think that code has been there the first time I looked at diesel. I'd absolutely continue to use the abbreviations. `F, S, W` by themselves are not that intuitive, but if you have ever seen a SQL query, they totally make sense, and it's not worth the code bloat.

_Maybe_, once I got to `Of`, I'd've switched everything to two-letter names, though (`Se, Fr, Di, Wh, Or, Li, Of, Gr`).